assets/css/style.css

```diff
@@ -0,0 +1,5 @@
+h1 {
+  border: 1px solid blue;
+  box-shadow: 0 0 5px red;
+  background-color: silver;
+}
```

Comment:
Yo dawg, could you put a newline at the end of this file?

Reply:
Fine whatever.  H8U.